packages/react/src/components/Avatar/Avatar.mdx

```diff
@@ -0,0 +1,75 @@
+import { Meta, Canvas, Controls, Primary } from '@storybook/blocks';
+
+import * as AvatarStories from './Avatar.stories';
+
+<Meta of={AvatarStories} />
+
+# Avatar
+
+`Avatar` er en komponent som viser et bilde, initialer eller ikon for en bruker eller profil.
```

Comment:
Were we supposed to add info about recommendation for initals only being 2 characters?